storage/InsertDestination.cpp

```diff
@@ -60,25 +60,28 @@ InsertDestination::InsertDestination(const CatalogRelationSchema &relation,
                                      const StorageBlockLayout *layout,
                                      StorageManager *storage_manager,
                                      const std::size_t relational_op_index,
+                                     const std::size_t query_id,
```

Comment:
I suggest to move `query_id` before `relational_op_index` based on a top-down relations of things.